example/JSON/CalcCT.json

```
@@ -0,0 +1,13 @@
+{
+    "firstLine": "2",
+    "lastLine": "2",
+    "input": "./example/C2HDM_Input.dat",
```

Input file not found because of missing dot:
```suggestion
    "input": "../example/C2HDM_Input.dat",
```